README.md

```diff
@@ -56,6 +57,17 @@ The promise will reject if the stream emits an error.
 
 The promise will reject if it is canceled.
 
+```php
+$stream = accessSomeToLargeStream();
+
+Stream\buffer($stream, 1024)->then(function ($contents) {
+    var_dump(json_decode($contents));
+}, function ($error) {
+    // Reaching here when the stream buffer goes above the max size.
+    // (In this example that is 1024 bytes.)
```

Comment:
Note that the promise will also be rejected when a stream `error` is emitted or when the promise is cancelled.